### PR TITLE
Create pasteup's CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Throw in some docs on how to release pasteup as well as state the goals of pasteup itself as a library!

## Why?
Releasing is a bit of a chore and @zeftilldeath asked me to document it and I am but a humble servant

### CODE REVIEW

File: packages/pasteup/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+After your PR gets approved, but before merging run
+
+```
+make release-pasteup
+```
+This will ask you to up the version of pasteup. We use (semver)[https://semver.org] and you can check a quick reference below.
+
+That's it! Pasteup is now in [NPM](https://www.npmjs.com/package/@guardian/pasteup). There's a couple loose ends we need to deal with to tidy everything up:
+
+Your branch witll have a commit with a change to pasteup's `package.json` reflecting the new version. Right now we use alpha versions so you have to update the dependency version in `package.json` in `guui` and `frontend` too. With that done, push your changes to the packages & merge.
```
Comment (SiAdcock) [inline packages/pasteup/CONTRIBUTING.md]: Type here: `witll`

File: packages/pasteup/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Your changes will ripple to dotcom-rendering immediately. For other projects you must notify other consumers of the library of the updated version. At the moment that includes:
+
+- support-frontend (reader revenue teams)
+
+TBD: Should we create an email address? Do we have one?
+TBD: Should we release via riffraff or travis? What are the downsides? How hard is this to set up?
+
+
+# Versioning rules of thumb
+Given the cross-cutting nature of this project are committed to avoid breaking changes as that would erode trust from potential consumers:
```
Comment (SiAdcock) [inline packages/pasteup/CONTRIBUTING.md]: > Given the cross-cutting nature of this project, **we** are committed

File: packages/pasteup/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+That's it! Pasteup is now in [NPM](https://www.npmjs.com/package/@guardian/pasteup). There's a couple loose ends we need to deal with to tidy everything up:
+
+Your branch witll have a commit with a change to pasteup's `package.json` reflecting the new version. Right now we use alpha versions so you have to update the dependency version in `package.json` in `guui` and `frontend` too. With that done, push your changes to the packages & merge.
+
+Your changes will ripple to dotcom-rendering immediately. For other projects you must notify other consumers of the library of the updated version. At the moment that includes:
+
+- support-frontend (reader revenue teams)
+
+TBD: Should we create an email address? Do we have one?
+TBD: Should we release via riffraff or travis? What are the downsides? How hard is this to set up?
```
Comment (GHaberis) [inline packages/pasteup/CONTRIBUTING.md]: https://github.com/guardian/mobile-apps-article-templates publish the templates package to NPM as part of CI. They're building on teamcity and pushing to npm.

File: packages/pasteup/CONTRIBUTING.md (hunk 1 above)
Comment (SiAdcock) [reply packages/pasteup/CONTRIBUTING.md]:
```suggestion
Your branch will have a commit with a change to pasteup's `package.json` reflecting the new version. Right now we use alpha versions so you have to update the dependency version in `package.json` in `guui` and `frontend` too. With that done, push your changes to the packages & merge.
```

File: packages/pasteup/CONTRIBUTING.md (hunk 2 above)
Comment (SiAdcock) [reply packages/pasteup/CONTRIBUTING.md]:
```suggestion
Given the cross-cutting nature of this project, we are committed to avoiding breaking changes as that would erode trust from potential consumers:
```

File: packages/pasteup/CONTRIBUTING.md (hunk 3 above)
Comment (AWare) [reply packages/pasteup/CONTRIBUTING.md]: I can help set this up.

Comment (walaura) [reply packages/pasteup/CONTRIBUTING.md]: omg that would be pretty awesome??

**Pull request merged**